packages/transport/test/testHelpers/AdminApiClient.ts

```diff
@@ -0,0 +1,40 @@
+import axios, { Axios } from "axios";
+import { AccountController, CoreId, IdentityDeletionProcess } from "../../src";
+import { TestUtil } from "./TestUtil";
+
+export class AdminApiClient {
+    private static adminClient: Axios | undefined;
+
+    public static async getBackboneAdminApiClient(): Promise<Axios> {
+        if (AdminApiClient.adminClient) {
+            return AdminApiClient.adminClient;
+        }
+        const adminAPIBaseUrl = process.env.NMSHD_TEST_BASEURL_ADMIN_API!;
+        if (!adminAPIBaseUrl) throw new Error("Missing environment variable NMSHD_TEST_BASEURL_ADMIN_API");
+        const csrf = await axios.get(`${adminAPIBaseUrl}/api/v1/xsrf`);
+        AdminApiClient.adminClient = axios.create({
+            baseURL: adminAPIBaseUrl,
+            headers: {
+                /* eslint-disable @typescript-eslint/naming-convention */
+                cookie: csrf.headers["set-cookie"],
+                "x-xsrf-token": csrf.data,
+                "x-api-key": process.env.NMSHD_TEST_ADMIN_API_KEY!
```

Comment:
Since we throw an error if the above envvar `NMSHD_TEST_BASEURL_ADMIN_API` doesn't exist, I'd say we do th same for `NMSHD_TEST_ADMIN_API_KEY`.

Reply:
Na the API key is localy not required only optional